circleci/integration-test

```diff
@@ -73,6 +79,34 @@ fi
 
 echo "Job ID: $JOB_ID"
 
+install_awscli
+# aws login.
+AWS_S3_PROFILE=oidc-s3-profile
+if [[ -v OIDC_S3_UPLOAD_ROLE ]]; then
+  echo "Logging into AWS using role credentials...."
+  assume_role_with_web_identity $OIDC_S3_UPLOAD_ROLE $AWS_S3_PROFILE 
+fi
+
+download_test_artifacts(){
+  echo "Downloading test artifacts"
+  mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+  for workflowID in $(cat integration-tests.out | jq -r '.testWorkflowIds[]');
+  do
+    S3_BUCKET_URL="s3://frontend-to-end-tester-files-dev/${workflowID}/artifacts.zip"
+    DEST="${CIRCLE_ARTIFACTS}/${workflowID}/artifacts.zip"
+    echo "  Source: $S3_BUCKET_URL"
+    echo "  Desination: $DEST"
+    if [[ -v OIDC_S3_UPLOAD_ROLE ]]; then
+        echo "Downloading files from S3 using profile ${AWS_S3_PROFILE}" 
+        aws s3 cp --profile $AWS_S3_PROFILE  $S3_BUCKET_URL $DEST
+    else
+        echo "Downloading files from S3 using static credentials" 
+        aws s3 cp $S3_BUCKET_URL $CIRCLE_ARTIFACTS
+    fi
+    unzip $DEST test-results.xml -d $CIRCLE_TEST_REPORTS/${workflowID}
```

Comment:
I think there might be a bug here. 

Say if "tmp" is the name of "CIRCLE_ARTIFACTS", so dest would look like: `tmp/workflow123/artifacts.zip`

going through the if statement would mean `tmp/workflow123/artifacts.zip` is the file location, but going through the else would mean `tmp/artifacts.zip` becomes the file location i think. which means running unzip in the else case wouldn't work?

Reply:
Good catch, fixed in https://github.com/Clever/ci-scripts/pull/130/commits/359a8a9fd42e3d78834d143105446f2c28092054